[test-plugins-build.sh] Enable kubewarden test after node 24 migration

PR #16847 disables the Kubewarden extension tests. Re-enable the tests after a shell version targeting node 24 is released and `kubewarding-ui` is updated. 


## Additional details 

`kubewarden-ui` fails with the following error:

```
Installing dependencies for kubewarden-ui

yarn install v1.22.22
[1/5] Validating package.json...
[2/5] Resolving packages...
(node:4124) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
(Use `node --trace-deprecation ...` to show where the warning was created)
[3/5] Fetching packages...
warning bare-fs@4.5.2: The engine "bare" appears to be invalid.
warning bare-os@3.6.2: The engine "bare" appears to be invalid.
warning chart.js@4.5.1: The engine "pnpm" appears to be invalid.
error @achrinza/node-ipc@9.2.9: The engine "node" is incompatible with this module. Expected version "8 || 9 || 10 || 11 || 12 || 13 || 14 || 15 || 16 || 17 || 18 || 19 || 20 || 21 || 22". Got "24.14.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/shell/scripts/test-plugins-build.sh
+++ b/shell/scripts/test-plugins-build.sh
@@ -243,6 +243,6 @@ clone_repo_test_extension_build "neuvector" "manager-ext" "neuvector-ui-ext"
 clone_repo_test_extension_build "harvester" "harvester-ui-extension" "harvester"
 clone_repo_test_extension_build "rancher" "ali-ui" "ali"
 clone_repo_test_extension_build "rancher" "virtual-clusters-ui" "virtual-clusters"
-clone_repo_test_extension_build "rancher" "rancher-ai-ui" "rancher-ai-ui"
+# clone_repo_test_extension_build "rancher" "rancher-ai-ui" "rancher-ai-ui"
 
 echo "All done"

--- a/shell/scripts/test-plugins-build.sh
+++ b/shell/scripts/test-plugins-build.sh
@@ -237,7 +237,7 @@ function clone_repo_test_extension_build() {
 # Don't forget to add the unit tests exception to clone_repo_test_extension_build function if a new extension has those
 # TODO: ISSUE #16858 - Reenable the tests as packages migrate to node version 24
 clone_repo_test_extension_build "rancher" "kubewarden-ui" "kubewarden"
-clone_repo_test_extension_build "rancher" "elemental-ui" "elemental"
+# clone_repo_test_extension_build "rancher" "elemental-ui" "elemental"
 clone_repo_test_extension_build "neuvector" "manager-ext" "neuvector-ui-ext"
 clone_repo_test_extension_build "StackVista" "rancher-extension-stackstate" "observability"
 clone_repo_test_extension_build "harvester" "harvester-ui-extension" "harvester"

--- a/shell/scripts/test-plugins-build.sh
+++ b/shell/scripts/test-plugins-build.sh
@@ -236,10 +236,13 @@ function clone_repo_test_extension_build() {
 # Here we just add the extension that we want to include as a check (all our official extensions should be included here)
 # Don't forget to add the unit tests exception to clone_repo_test_extension_build function if a new extension has those
 # TODO: ISSUE #16858 - Reenable the tests as packages migrate to node version 24
-# clone_repo_test_extension_build "rancher" "kubewarden-ui" "kubewarden"
-# clone_repo_test_extension_build "rancher" "elemental-ui" "elemental"
+clone_repo_test_extension_build "rancher" "kubewarden-ui" "kubewarden"
+clone_repo_test_extension_build "rancher" "elemental-ui" "elemental"
 clone_repo_test_extension_build "neuvector" "manager-ext" "neuvector-ui-ext"
-# clone_repo_test_extension_build "StackVista" "rancher-extension-stackstate" "observability"
-# clone_repo_test_extension_build "harvester" "harvester-ui-extension" "harvester"
+clone_repo_test_extension_build "StackVista" "rancher-extension-stackstate" "observability"
+clone_repo_test_extension_build "harvester" "harvester-ui-extension" "harvester"
+clone_repo_test_extension_build "rancher" "ali-ui" "ali"
+clone_repo_test_extension_build "rancher" "virtual-clusters-ui" "virtual-clusters"
+clone_repo_test_extension_build "rancher" "rancher-ai-ui" "rancher-ai-ui"
 
 echo "All done"

--- a/shell/scripts/test-plugins-build.sh
+++ b/shell/scripts/test-plugins-build.sh
@@ -239,7 +239,7 @@ function clone_repo_test_extension_build() {
 clone_repo_test_extension_build "rancher" "kubewarden-ui" "kubewarden"
 # clone_repo_test_extension_build "rancher" "elemental-ui" "elemental"
 clone_repo_test_extension_build "neuvector" "manager-ext" "neuvector-ui-ext"
-clone_repo_test_extension_build "StackVista" "rancher-extension-stackstate" "observability"
+# clone_repo_test_extension_build "StackVista" "rancher-extension-stackstate" "observability"
 clone_repo_test_extension_build "harvester" "harvester-ui-extension" "harvester"
 clone_repo_test_extension_build "rancher" "ali-ui" "ali"
 clone_repo_test_extension_build "rancher" "virtual-clusters-ui" "virtual-clusters"

